specs/www.rst

```diff
@@ -22,6 +22,7 @@ This spec has had the following versions:
 * ``2.2``: Allow ``None`` in the second item of ``server`` scope value.
 * ``2.3``: Added the ``reason`` key to the WebSocket close event.
 * ``2.4``: Calling ``send()`` on a closed connection should raise an error
+* ``2.5``: Added the ``reason`` key to the WebSocket disconnect event.
```

Comment:
Does this need a bump on the spec?

Reply:
Not for me to say, but I saw it was done similarly for 2.3. Can remove it if not needed :)

Reply:
Yes, any spec change should minor bump if it's backwards-compatible (like this) and a major bump if it is not.

Reply:
You did miss out updating the main version number though - we'll need that bumped too.

Reply:
> You did miss out updating the main version number though - we'll need that bumped too.

Done 👍